src/dmd/frontend.h

```diff
@@ -3424,89 +3424,6 @@ class TemplateMixin final : public TemplateInstance
 
 extern void genCppHdrFiles(Array<Module* >& ms);
 
-class ToCppBuffer final : public Visitor
```

Comment:
This declaration is now longer emitted because it is `private`

Reply:
I noticed that some of the class's methods are made `private`. Is that really necessary, since the class is `private`?

Reply:
No. I just marked the few remaining methods as `private` s.t. it's consistently used for all internal methods.